Documentation/release-notes/dx-deprecation.md

```diff
@@ -0,0 +1,6 @@
+  * warning XA1023: Using the DX Dex Compiler is deprecated ... Google
+    has deprecated the DX Dex Compiler. DX will be [completely
+    removed][dx] in a future release of the Android SDK and Android
+    Studio.
+
+[dx]: https://android-developers.googleblog.com/2020/02/the-path-to-dx-deprecation.html
```

Comment:
Excellent that this has a release note included right from the get-go. Thanks much on that!